docs/content/documentation/use.md

```diff
@@ -244,6 +244,9 @@ Determine the number of rows fetched in `ResultSet` by one fetch with trip to th
 Limiting the number of rows are fetch with each trip to the database allow avoids unnecessary memory consumption and as a consequence `OutOfMemoryError` .
 The default is zero, meaning that `ResultSet` will fetch all rows at once. Must be > 0.
 
+* **`defaultQueryTimeout (`*int*`)`** *Default `0`*\
+The default timeout value in seconds that the driver will wait for a query to execute.
```

Comment:
We'd probably rephrase it so it mentions `Statement#setQueryTimeout` so the users don't confuse it with PostgreSQL queries. We should probably clarify the value of `0` means going with no limit